share/sources/org.biorxiv/source.yaml

```diff
@@ -22,6 +22,15 @@ configs:
       namespaces:
           'http://purl.org/rss/1.0/': null
           'http://purl.org/dc/elements/1.1/': dc
+- base_url: http://biorxiv.org/search/
+  disabled: false
+  earliest_date: 2013-11-01
+  harvester: org.biorxiv.html
+  harvester_kwargs: {}
+  label: org.biorxiv.html
+  rate_limit_allowance: 1
+  rate_limit_period: 10
```

Comment:
I just realized that I had changed this back. I couldn't stand wait that long locally.